package.json

```diff
@@ -40,5 +40,6 @@
     "scripts": {
         "test": "nyc mocha tests/rules/",
         "lint": "eslint ."
-    }
+    },
+    "files": ["index.js", "lib", "LICENSE", "docs", "SECURITY.md", "README.md"]
```

Comment:
@mozfreddyb should we add also SCHEMA.md? it seems to be referred from docs/rules/customization.md which is explicitly included in the files listed here.

Reply:
Good catch! Update coming.

Reply:
New commit also sorts entries.